packages/core/src/Field/Field.js

```diff
@@ -1,25 +1,56 @@
-import React from 'react'
 import propTypes from '@dhis2/prop-types'
+import React from 'react'
 
-import { spacers } from '@dhis2/ui-constants'
-;('') // TODO: https://github.com/jsdoc/jsdoc/issues/1718
+import { sharedPropTypes, spacers } from '@dhis2/ui-constants'
+import { Label } from '../Label/Label.js'
+import { Help } from '../Help/Help.js'
+import { Box } from '../Box/Box.js'
 
 /**
  * @module
  * @param {Field.PropTypes} props
  * @returns {React.Component}
- * @example import { Field } from @dhis2/ui-core
- * @see Live demo: {@link /demo/?path=/story/field--default|Storybook}
+ *
+ * @example import { Field } from '@dhis2/ui'
+ *
+ * @example import { Field } from '@dhis2/ui-core'
  */
-const Field = ({ children, className, dataTest }) => (
-    <div className={className} data-test={dataTest}>
-        {children}
-        <style jsx>{`
-            div {
-                margin: 0 0 ${spacers.dp16};
-            }
-        `}</style>
-    </div>
+const Field = ({
+    children,
+    disabled,
+    className,
+    helpText,
+    label,
+    name,
+    validationText,
+    required,
+    dataTest,
+    valid,
+    error,
+    warning,
+}) => (
+    <Box className={className} dataTest={dataTest}>
+        {label && (
+            <Label required={required} disabled={disabled} htmlFor={name}>
```

Comment:
We could set the dataTest here similar to for the helpText and the validationText: 

```
dataTest={`${dataTest}-label`}
```

same with the box: 

```
// Or -children, or sth like that
dataTest={`${dataTest}-content`}
```

Reply:
48306a5

Thanks!